frontend/packages/data-portal/app/components/Run/TomogramTable.tsx

```diff
@@ -152,29 +153,28 @@ export function TomogramsTable() {
         cell: ({ row: { original } }) => (
           <TableCell width={TomogramTableWidths.actions}>
             <div className="flex flex-col gap-sds-xs items-start">
-              {original.is_canonical &&
-                original.neuroglancer_config != null && (
-                  <ViewTomogramButton
-                    tomogramId={original.id.toString()}
-                    neuroglancerConfig={original.neuroglancer_config}
-                    buttonProps={{
-                      sdsStyle: 'square',
-                      sdsType: 'primary',
-                      className: '!text-sds-body-xxs !h-sds-icon-xl',
-                      startIcon: (
-                        <Icon sdsIcon="Cube" sdsType="button" sdsSize="xs" />
-                      ),
-                    }}
-                    tooltipPlacement="top"
-                    event={{
-                      datasetId: run.dataset.id,
-                      organism: run.dataset.organism_name ?? 'None',
-                      runId: run.id,
-                      tomogramId: original.id,
-                      type: 'tomogram',
-                    }}
-                  />
-                )}
+              {original.neuroglancerConfig != null && ( // TODO(bchu): Check it's either isStandardized or isAuthorSubmitted
+                <ViewTomogramButton
+                  tomogramId={original.id.toString()}
+                  neuroglancerConfig={original.neuroglancerConfig}
+                  buttonProps={{
+                    sdsStyle: 'square',
+                    sdsType: 'primary',
+                    className: '!text-sds-body-xxs !h-sds-icon-xl',
+                    startIcon: (
+                      <Icon sdsIcon="Cube" sdsType="button" sdsSize="xs" />
+                    ),
+                  }}
+                  tooltipPlacement="top"
+                  event={{
+                    datasetId: run.dataset.id,
+                    organism: run.dataset.organism_name ?? 'None',
+                    runId: run.id,
```

Comment:
this one is still snake case?

Reply:
Yeah the `run` query isn't being used yet. The code is in a confusing state rn and the sooner we can finish the migration the better 😭